sdk/src/main/java/com/karhoo/sdk/api/model/QuoteList.kt

```diff
@@ -5,4 +5,6 @@ import kotlinx.android.parcel.Parcelize
 
 @Parcelize
 data class QuoteList(val categories: Map<String, List<Quote>>,
-                     val id: QuoteId) : Parcelable
+                     val id: QuoteId,
+                     val status: String?,
+                     val validity: Int) : Parcelable
```

Comment:
can we add `AvailabilityVehicle` as well? ... 
do you want to do it once you verified these changes are working or not?

Reply:
I would really like to avoid adding that for now. While useful for this object, better for a task of its own.